README.md

```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
 #### Updating CRA
 
 When updating Create React App it's important to take a look at the `optimization.minimizer` values in the webpack config and then update the config in `craco.config.js`. After copying over any updates be sure to re-introduce the `terserOptions.mangle.reserved` key and values in the newly updated config.
+
+## Releasing the Dashboard
+
+To complete QA and release the Dashboard will take aproximately 2h of total time, 1h of which is an automated process at the end.
+
+- [Release prep](#release-prep)
+- [Generate a release tarball for Juju](#generate-a-release-tarball-for-juju)
+- [Release to jaas.ai](#release-to-jaas.ai)
+- [Release to Juju Simplestreams](#release-to-juju-simplestreams)
+- [Create an announcement on the Juju Discourse](#create-an-announcement-on-the-juju-discourse)
+
+### Release prep
+
+To prepare to release:
+
+- Clone a new copy of the repository to remove the chance of any uncommitted artifacts ending up in the release.
+- Thoroughly [QA the Dashboard](#qa'ing-the-dashboard) following the [Release tests](#release-tests)
```

Comment:
This is a big ask on s lightweight release. They should of course QA it through.

Reply:
Yeah I feel like over time we'll be able to enhance our automated testing infrastructure and these steps (which I haven't actually written yet) will be dramatically reduced. But the issue right now is that none of that exists, and these releases will go out into peoples production environments so we want to make sure they are thoroughly qa'd. So I'm err'ing on the side of caution in this document around these steps.